lca8s8-d1e52/cod-dlc9qq_lca8s8-d1e52.xml

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader>
+    <fileDesc>
+      <titleStmt>
+        <title>Caput 1 [Bnf 12926 Transcription]</title>
+        <author ref="#per-bdlcbd">Bartolome de las Casas</author>
+        <respStmt>
+          <name ref="#a1wilson">Andrew L. Wilson</name>
+          <resp>Transcription Editor</resp>
+          <resp>TEI Encoder</resp>
+        </respStmt>
+      </titleStmt>
+      <editionStmt>
+        <edition n="0.0.0-dev">
+          <title>Caput 1 [Bnf 12926 Transcription]</title>
+          <date when="2024-07-09">July 9, 2024</date>
+        </edition>
+      </editionStmt>
+      <publicationStmt>
+        <authority>SCTA</authority>
+        <availability status="free">
+          <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref></p>
+        </availability>
+      </publicationStmt>
+      <sourceDesc>
+        <listWit>
+          <witness xml:id="P" n="cod-dlc9qq">Bnf Latin 12926</witness>
+        </listWit>
+      </sourceDesc>
+    </fileDesc>
+    <encodingDesc>
+      <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+      <editorialDecl>
+        <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+          guidelines for a diplomatic edition. </p>
+      </editorialDecl>
+    </encodingDesc>
+    <revisionDesc status="draft">
+      <listChange>
+        <change when="2024-07-09" status="draft" n="0.0.0">
+          <p>Created file for the first time.</p>
+        </change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#P" n="11-r"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="lca8s8-d1e52">
+		<head id="P">Caput 1 [Bnf 12926 Transcription]</head>
+		<p>
+			<lb ed="#P" n="P"/>Adversvs persecvtores, et calum<lb ed="#P" break="no" n="P"/>niatores 
+			gentivm novi orbis
+			<lb ed="#P" n="P"/>ad oceanum reperti apolo<lb ed="#P" break="no" n="P"/>gia:
+			admodum Reverendissimi domini d. fratris Bartholomei cas<lb ed="#P" break="no" n="P"/>aus 
+			episcopi quondam
+			<lb ed="#P" n="P"/>chiapensis sacri ordinis predicatorum.
+		</p>
+		<p>
+			<lb ed="#P" n="P"/>Capvt primum
+		</p>
+		<p>
+			<lb ed="#P" n="P"/>Quicumque vel voce vel scriptis docent noui
```

Comment:
We don't need a n="P" here. I think that might be a misunderstanding. 

The `@n` attribute will hold the line number, but we will add that automatically when the PR is accepted. 

So you should just put <lb ed="#P"/>

I guess in this in case you should just leave it as is and I will remove all the n="P" with a find a replace. Or you can do that if you feel comfortable doing that.

```suggestion
			<lb ed="#P"/>Quicumque vel voce vel scriptis docent noui
```

Reply:
I can go ahead and remove these (easy search/replace) on my end and recommit. Should I open a new pull request? I'm not sure how to proceed.